base_classes/NXfabrication.nxdl.xml

```diff
@@ -0,0 +1,70 @@
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<!--
+# NeXus - Neutron and X-ray Common Data Format
+# 
+# Copyright (C) 2014-2024 NeXus International Advisory Committee (NIAC)
+# 
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+# For further information, see http://www.nexusformat.org
+-->
+<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="base" type="group" name="NXfabrication" extends="NXobject" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+    <doc>
+         Details about a component as it is defined by its manufacturer.
+    </doc>
+    <field name="vendor" type="NX_CHAR">
+        <doc>
+             Company name of the manufacturer.
+        </doc>
+    </field>
+    <field name="model" type="NX_CHAR">
+        <doc>
+             Version or model of the component named by the manufacturer.
+        </doc>
+        <attribute name="version" type="NX_CHAR">
+            <doc>
+                 If different versions exist are possible, the value in this field should be made
+                 specific enough to resolve the version.
+            </doc>
+        </attribute>
+    </field>
+    <group name="identifier" type="NXidentifier">
+        <doc>
+             Ideally, (globally) unique persistent identifier. This may contain e.g. a hash
+             identifier of the component.
+        </doc>
+    </group>
+    <field name="serial_number" type="NX_CHAR">
+        <doc>
+             Serial number of the component.
+        </doc>
+    </field>
+    <field name="construction_year" type="NX_DATE_TIME">
+        <doc>
+             Datetime of components initial construction. This refers to the date of
+             first measurement after new construction or to the relocation date,
+             if it describes a multicomponent/custom-build setup.
+             Should be a ISO8601 date/time stamp. It is recommended to add an explicit time zone,
+             otherwise the local time zone is assumed per ISO8601.
+        </doc>
+    </field>
+    <field name="capability" type="NX_CHAR">
+        <doc>
+             Free-text list with eventually multiple terms of
+             functionalities which the component offers.
```

Comment:
Free-text list of functionalities which the component offers.

Reply:
done